src/wp-content/themes/twentytwentyfour/patterns/page-home-blogging.php

```diff
@@ -22,7 +22,7 @@
 
 		<!-- wp:column {"width":"60%"} -->
 		<div class="wp-block-column" style="flex-basis:60%">
-			<!-- wp:query {"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false}} -->
+			<!-- wp:query {"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true}} -->
```

Comment:
What's the rationale for this change? I'm only asking because it wasn't part of https://github.com/WordPress/twentytwentyfour/pull/706. Is this being fixed simply because it's another blog pattern that currently ignores the main query? Is this pattern used in any of the theme's default templates, or is it just a pattern that site owners can use themselves anywhere, without any default usage?

Reply:
This was a separate fix [here](https://github.com/WordPress/twentytwentyfour/pull/708). We offer this pattern as a replacement for the one on the home page via the [template swappingflow](https://github.com/WordPress/gutenberg/pull/54609), so it makes sense that it inherits the query so the template behaves as expected.